### ISSUE TITLE
Export to LaTeX with pandoc

### ISSUE THREAD

**Issue by dpo** [shd101wyy/markdown-preview-enhanced#647]
I'm trying to debug an issue when exporting to PDF with Pandoc via a LaTeX template. In order to have access to the intermediate LaTeX source, I use the following in the YAML header:
```yaml
output:
  custom_document:
    path: mydoc.tex
    pandoc_args: [
      "-r",  "markdown+tex_math_single_backslash+simple_tables+table_captions+yaml_metadata_block",
      "-t", "latex",
      "-S",
      "--template=/path/to/my/latex.template",
      "-v", "natbib"
    ]
```
On exporting with pandoc, the popup messages say that `mydoc.tex` was created but it's nowhere to be found.
The console shows the following:

![screen shot 2017-10-09 at 13 34 06](https://user-images.githubusercontent.com/38760/31350820-904704d8-acf6-11e7-95aa-fcbe682a62c8.png)

Any idea what I'm doing wrong?

Thanks!

**Comment by shd101wyy** [comment]
```markdown
---
output:
  custom_document:
    pandoc_args: [
      "-r",  "markdown+tex_math_single_backslash+simple_tables+table_captions+yaml_metadata_block",
      "-t", "latex",
      "-S",
      "--template=/path/to/my/latex.template",
      "-v", "natbib",
      "-o", "mydoc.tex"
    ]
---
```

`path` only works with MPE builtin support file types. So you need to add `-o mydoc.tex` to `pandoc_args`

**Comment by dpo** [comment]
`path` works when I export to html with pandoc, even though it's not a builtin type. If I specify `-o` in the `pandoc_args` and remove `path`, MPE complains that `path` must be specified. If I specify both `path` and `-o`, MPE doesn't complain but the file isn't created either.

Any ideas?

**Comment by dpo** [comment]
Sorry, my error. The pandoc option `-v` should really be `-V` (capital). It would be great to report an error in a case like this though. For some reason, MPE is happy.

**Comment by shd101wyy** [comment]
Yes, you are correct. You should just use `path` instead of writing `-o` yourself. 
I just tried myself with the following markdown and it works:  

```markdown
---
output:
  custom_document:
    path: my_doc.tex
    pandoc_args: [
      "-r",  "markdown+tex_math_single_backslash+simple_tables+table_captions+yaml_metadata_block",
      "-t", "latex",
      "-S",
    ]
---

# Hi There
```

So I think it is your front-matter that caused the problem.
